common/arrow/Cargo.toml

```diff
@@ -7,6 +7,7 @@ publish = false
 edition = "2018"
 
 [features]
+default = ["simd"]
```

Comment:
Hmm, why this work? :dancers:

Reply:
By default, all features are disabled unless explicitly enabled. This can be changed by specifying the default feature.

From https://doc.rust-lang.org/cargo/reference/features.html

Reply:
Thank you